packages/embark-solidity-tests/src/lib/reporter.js

```diff
@@ -3,17 +3,17 @@ class Reporter {
     this.reporter = reporter;
   }
 
-  report(payload) {
+  report(err, payload) {
```

Comment:
Should we check the `err`?

Reply:
We don't have to, as [an error never gets passed](https://github.com/ethereum/remix/blob/master/remix-tests/src/testRunner.ts). However, I should really rename it to `_err` to show that we never reference it. Good call.